src/zenml/models/v2/core/pipeline_run.py

```diff
@@ -115,6 +115,11 @@ class PipelineRunRequest(WorkspaceScopedRequest):
         default=None,
         title="Tags of the pipeline run.",
     )
+    configured_model_version_id: Optional[UUID] = Field(
```

Comment:
Hm, it will be the same, but the `model_version` is a populated `ModelVersionResponse` using the `configured_model_version_id` to lookup for the model version. Is this somehow wrong?

Reply:
I would not say that it is wrong but it feels like the effort is duplicated. In a regular case (where the model version is in the `Resources`), you would always have access to the id of the model version (regardless of hydration) through `pipeline_run.model_version.id` or vice versa.

Reply:
It is also fine to leave it like this btw. If we want we can improve this later on, resolving the issue here.